### PR TITLE
Implement DataFrame.swapaxes

### PR DESCRIPTION
**Pull request by xinrong-databricks**
Implement DataFrame.swapaxes
```
        >>> kdf = ks.DataFrame(
        ...     [[1, 2, 3], [4, 5, 6], [7, 8, 9]], index=['x', 'y', 'z'], columns=['a', 'b', 'c']
        ... )
        >>> kdf
           a  b  c
        x  1  2  3
        y  4  5  6
        z  7  8  9
        >>> kdf.swapaxes()
           x  y  z
        a  1  4  7
        b  2  5  8
        c  3  6  9
        >>> kdf.swapaxes(i=1, j=0)
           x  y  z
        a  1  4  7
        b  2  5  8
        c  3  6  9
        >>> kdf.swapaxes(i=1, j=1)
           a  b  c
        x  1  2  3
        y  4  5  6
        z  7  8  9
```

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -6448,6 +6448,53 @@ def swaplevel(self, i=-2, j=-1, axis=0) -> "DataFrame":
 
         return DataFrame(internal)
 
+    def swapaxes(
+        self, i: Union[str, int] = 0, j: Union[str, int] = 1, copy: bool = True
+    ) -> "DataFrame":
+        """
+        Interchange axes and swap values axes appropriately.
```
Comment (ueshin) [inline databricks/koalas/frame.py:6455]: As this is almost the same as `transpose`, shall we add a note as well? See https://github.com/databricks/koalas/blob/53812b8a2b7ffeaaaed66ea11ccf92d09a6cee05/databricks/koalas/frame.py#L1978-L1991

**Review by ueshin — APPROVED** [review]
Otherwise, LGTM.

File: databricks/koalas/frame.py (hunk 1 above)
Comment (xinrong-databricks) [reply databricks/koalas/frame.py:6455]: Sounds good! Added.

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -6448,6 +6448,68 @@ def swaplevel(self, i=-2, j=-1, axis=0) -> "DataFrame":
...
+
+                >>> from databricks.koalas.config import option_context
+                >>> with option_context('compute.max_rows', 1000):  # doctest: +NORMALIZE_WHITESPACE
+                ...     ks.DataFrame({'a': range(1001)}).swapaxes()
+                Traceback (most recent call last):
+                  ...
+                ValueError: Current DataFrame has more then the given limit 1000 rows.
+                Please set 'compute.max_rows' by using 'databricks.koalas.config.set_option'
+                to retrieve to retrieve more than 1000 rows. Note that, before changing the
+                'compute.max_rows', this operation is considerably expensive.
```
Comment (ueshin) [inline databricks/koalas/frame.py:6479]: Could you move this to above `Parameters`? This place breaks the doc format:

Screen Shot 2020-12-03 at 4 51 56 PM

Comment (ueshin) [reply databricks/koalas/frame.py:6470]: `./dev/lint-python` generates the HTML version, or `cd docs/; make html`?

Comment (xinrong-databricks) [reply databricks/koalas/frame.py:6470]: Got it! Thanks!

Comment (xinrong-databricks) [reply databricks/koalas/frame.py:6470]: image

**Pull request merged**